### PR TITLE
JNG-4604 Environment variables and sequence tests

### PR DESCRIPTION
**Pull request by bencegelei**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4604" title="JNG-4604" target="_blank">StoryJNG-4604</a>  Environment variables and sequence tests
  </td></table>
  <br />

### CODE REVIEW

**Review by robertcsakany — APPROVED** [review]

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+            "REQ-EXPR-007",
+            "REQ-EXPR-009",
+            "REQ-EXPR-012"
+    })
+    void testVerifyTheGetVariableShortcuts() {
+        EnvVars envVars = envVarsDao.create(EnvVars.builder().build());
+
+        LocalDate localDate = envVars.getCd().orElseThrow();
+        LocalDate localDate1 = envVars.getCd1().orElseThrow();
+        if (localDate.isBefore(localDate1)) {
```
Comment (noherczeg) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java:357]: This shouldn't be needed.

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java (hunk 1 above)
Comment (bencegelei) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java:357]: There is a slight possibility that different current dates get executed on different days.

Comment (noherczeg) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java:357]: Then we can re-run the job.

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java (hunk 1 above)
Comment (bencegelei) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/EnvironmentAndSequenceTest.java:357]: https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/155/commits/3e5fb4f6260449127ccac922517405bf14304939

**Pull request merged**